storybookjs/storybook
# add ability to skip cypress tests based on framework

Issue: N/A

## What I did

This PR adds a way to skip certain specs based on frameworks, such as skipping dynamic rendering spec on `vue3`, which does not have dynamic rendering.

The command uses `name` which is defined in [here like this](https://github.com/storybookjs/storybook/blob/next/lib/cli/src/repro-generators/configs.ts#L35).

## How to test

- Clone the repo
- Install the deps
- Spin up verdaccio: `yarn local-registry --port 6000 --open --publish`
- On another terminal, run `yarn test:e2e-framework` and select vue3
- Select addon-docs spec
- It should skip vue3, but should run for other frameworks

- [x] Does this need an update to the documentation? As a followup, we should have an "E2E testing" improvement story so we can iron out some frustrations when running stuff locally + add some guidance docs.

If your answer is yes to any of these, please make sure to include it in your PR.
